contracts/chain-adapters/l2/WithdrawalHelperBase.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.19;
```

Comment:
Reminder to set this to 0.8.0 if we're going to do the same with arbitrum

Reply:
Good point. https://github.com/across-protocol/contracts/pull/728/commits/b6cae1c347123855208db93f43a03cb0736ba15b